tests/utils/staggered_host_utils.cpp

```diff
@@ -166,7 +166,7 @@ void constructFatLongGaugeField(void **fatlink, void **longlink, int type, QudaP
       const complex<double> z = std::polar(1.0, phase);
       for (int dir = 0; dir < 4; ++dir) {
         for (int i = 0; i < V; ++i) {
-          for (int j = 0; j < gauge_site_size; j += 2) {
+          for (unsigned int j = 0; j < gauge_site_size; j += 2) {
```

Comment:
any particular reason for `unsigned int` here when `gauge_site_size` is `size_t`? Anyway, how about `auto` ?

Reply:
I am also wondering about the same thing here ...